pages/linux/rc-status.md

```diff
@@ -0,0 +1,32 @@
+# rc-status
+
+> Show status info about runlevels.
+> Part of OpenRC - see also `openrc`.
```

Comment:
```suggestion
> See also `openrc`.
```

Reply:
Hrm. I thought explicitly indicating that it's part of OpenRC was a good idea.